lib/mini_twitter_client.rb

```diff
@@ -0,0 +1,84 @@
+require 'faraday'
+require 'json'
+require_relative 'tweet'
+
+class Response
```

Comment:
You can keep every class in separate file, so in this case you can move it to `response.rb`